src/OperationalInsights/OperationalInsights/Workspaces/SetAzureOperationalInsightsWorkspaceCommand.cs

```diff
@@ -94,7 +97,8 @@ public override void ExecuteCmdlet()
                 RetentionInDays = RetentionInDays,
                 DailyQuotaGb = DailyQuotaGb,
                 ForceCmkForQuery = ForceCmkForQuery,
-                WsFeatures = new PSWorkspaceFeatures(DisableLocalAuth)
+                WsFeatures = new PSWorkspaceFeatures(DisableLocalAuth),
+                DefaultDataCollectionRuleResourceId = DefaultDataCollectionRuleResourceId,
```

Comment:
remove the last comma, please use your IDE to check these

Reply:
removed